CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Change
```

Comment:
Oh, noticed now that instead of "Changed" we have been adding "Change", do you think we can change all to "Changed" (to maintain the same verb tense as "Added" or "Fixed"). Thx

Reply:
Sure